### PR TITLE
Add some irrelevant files

### PR DESCRIPTION
**Pull request by kajinamit**
... to avoid running jobs unnecessarily.

As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [x] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

File: zuul.d/edpm.yaml [hunk 1]
```diff
@@ -19,6 +19,12 @@
     abstract: true
     irrelevant-files:
```
Comment (cjeanner) [inline zuul.d/edpm.yaml:20]: wondering if we should get "files" and run only if ci_framework/* is changed? Seems a bit more suitable.. WDYT?

Comment (kajinamit) [reply zuul.d/edpm.yaml:20]: This job template is used to run edpm job in idividual repo, and we have to run it in any logic change.

**Comment by raukadah** [comment]
recheck

**Review by raukadah — APPROVED** [review]
/approve

**Pull request merged**